systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -337,10 +337,11 @@
 ret{1} = zeros(16, nq);
 for i = 2 : nb
   body = bodies(i);
-  T_body_to_parent = T{body.parent} \ T{i};
+  %T_body_to_parent = T{body.parent} \ T{i};
+  T_body_to_parent = T{body.parent}^-1 * T{i};
```

Comment:
msspoly does not support \

-Andy S.


On Thu, Jul 30, 2015 at 11:07 PM, Twan Koolen <notifications@github.com>
wrote:

> In systems/plants/@RigidBodyManipulator/doKinematics.m
> <https://github.com/RobotLocomotion/drake/pull/1212#discussion_r35943632>:
>
> > @@ -337,10 +337,11 @@
> >  ret{1} = zeros(16, nq);
> >  for i = 2 : nb
> >    body = bodies(i);
> > -  T_body_to_parent = T{body.parent} \ T{i};
> > +  %T_body_to_parent = T{body.parent} \ T{i};
> > +  T_body_to_parent = T{body.parent}^-1 * T{i};
>
> Why was this necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1212/files#r35943632>.
>